modules/misc/qt.nix

```diff
@@ -8,15 +8,15 @@ let
     bb10bright = libsForQt5.qtstyleplugins;
     bb10dark = libsForQt5.qtstyleplugins;
     cleanlooks = libsForQt5.qtstyleplugins;
-    gtk2 = libsForQt5.qtstyleplugins;
+    gtk2 = [ libsForQt5.qtstyleplugins qt6Packages.qt6gtk2 ];
     motif = libsForQt5.qtstyleplugins;
     cde = libsForQt5.qtstyleplugins;
     plastique = libsForQt5.qtstyleplugins;
 
-    adwaita = adwaita-qt;
-    adwaita-dark = adwaita-qt;
-    adwaita-highcontrast = adwaita-qt;
-    adwaita-highcontrastinverse = adwaita-qt;
+    adwaita = [ adwaita-qt adwaita-qt6 ];
+    adwaita-dark = [ adwaita-qt adwaita-qt6 ];
+    adwaita-highcontrast = [ adwaita-qt adwaita-qt6 ];
+    adwaita-highcontrastinverse = [ adwaita-qt adwaita-qt6 ];
```

Comment:
A bit iffy on this, considering upstream is discontinued (https://github.com/FedoraQt/adwaita-qt#adwaita-qt-project-is-unmaintained-a-no-longer-actively-developed).

I also added it in #4312, but that was a few days before the notice was put up.

Reply:
I don't see a good reason to remove this unless we remove the package from nixpkgs itself.

I also recently added this same option in the NixOS module (see https://github.com/NixOS/nixpkgs/pull/262051), and it would be good to keep for feature parity.

If we eventually remove this from nixpkgs, I think we can remove it from here.

Reply:
Agreed.